elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -155,7 +155,7 @@ public void serialize(Object obj, JsonGenerator jsonGenerator,
      * @param opaqueUser the opaque user
      * @return Elide response object
      */
-    public ElideResponse get(String path, MultivaluedMap<String, String> queryParams, Object opaqueUser) {
+    public ElideResponse get(String path, MultivaluedMap<String, String> queryParams, User opaqueUser) {
```

Comment:
Should this be Principal?

Reply:
Elide deals with User directly so it can provide common checks (like role checks out of the box).